src/api/procedures/consumeAuthorizationRequests.ts

```diff
@@ -58,15 +58,16 @@ export async function prepareConsumeAuthorizationRequests(
 
     const idsPerType: Record<AllowedAuthType, [u64][]> = mapValues(typesToExtrinsics, () => []);
 
-    liveRequests.forEach(authRequest => {
+    for (const authRequest of liveRequests) {
```

Comment:
You can keep the forEach, push the assertion promise into an array in each iteration, and then await the array with Promise.all outside the forEach

Reply:
True, and then they will be in parallel if its a big batch.